server/opts.go

```diff
@@ -22,16 +22,18 @@ import (
 
 // ClusterOpts are options for clusters.
 type ClusterOpts struct {
-	Host           string      `json:"addr"`
-	Port           int         `json:"cluster_port"`
-	Username       string      `json:"-"`
-	Password       string      `json:"-"`
-	AuthTimeout    float64     `json:"auth_timeout"`
-	TLSTimeout     float64     `json:"-"`
-	TLSConfig      *tls.Config `json:"-"`
-	ListenStr      string      `json:"-"`
-	NoAdvertise    bool        `json:"-"`
-	ConnectRetries int         `json:"-"`
+	Host                string      `json:"addr"`
+	Port                int         `json:"cluster_port"`
+	Username            string      `json:"-"`
+	Password            string      `json:"-"`
+	AuthTimeout         float64     `json:"auth_timeout"`
+	TLSTimeout          float64     `json:"-"`
+	TLSConfig           *tls.Config `json:"-"`
+	ListenStr           string      `json:"-"`
+	ClientAdvertiseStr  string      `json:"-"`
+	ClusterAdvertiseStr string      `json:"-"`
```

Comment:
These should be stored as `[]string` and called `ClientAdvertiseAddrs` and `ClusterAdvertiseAddrs` IMO.

Reply:
Also, should `ClusterAdvertiseAddrs` be called `RouteAdvertiseAddrs` (assuming that's what it's used for)? I think that's the more common term we use.